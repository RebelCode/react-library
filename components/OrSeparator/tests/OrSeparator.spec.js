import React from "react";
import { render, cleanup } from "@testing-library/react";
import {OrSeparator} from "../index";

afterEach(cleanup);

describe("OrSeparator", () => {
  const Comp = (
    <OrSeparator />
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("OrSeparator")).toBeTruthy();
  });
});
