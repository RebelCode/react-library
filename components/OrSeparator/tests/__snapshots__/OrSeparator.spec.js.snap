// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrSeparator should take a snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="OrSeparator"
    class="container"
    style="display: flex; flex-flow: row nowrap; justify-content: center; align-items: center;"
  >
    <div
      class="line"
    />
    <span
      class="text"
    >
      OR
    </span>
    <div
      class="line"
    />
  </div>
</DocumentFragment>
`;
